docs/source/project-management/sdc-sit-plan.rst

```diff
@@ -0,0 +1,63 @@
+.. _sdc-sit-plan:
+
+SDC Software Integration Test (SIT) Plan
+========================================
+
+The SDC test plan is based on the `SOC Test Plan <https://lasp.colorado.edu/galaxy/display/IMAP/IMAP+SOC+Test+Plan>`_, but makes the below customizations to help fit the needs of SDC specific testing. Planned SIT dates and planned functionality tested can also be found in the linked SOC Test Plan.
+
+Software Inegration Test Timeline
+---------------------------------
+====================== =============================================================================================================================
+Timeline               Step
+====================== =============================================================================================================================
+Start of development   SDC creates a SIT procedures page and adds/updates test steps throughout the development process
+~4 weeks prior to test SDC begins to finalize test steps
+~2 weeks prior to test SDC conducts a dry run of the test procedures, makes any necessary fixes, test step updates
```

Comment:
you have "SDC ..." in all of these headings. Do you want to just put "Conduct a dry run ..." and leave that off?